src/utils/options.js

```diff
@@ -69,10 +69,17 @@ const unvotesDescription = `Specifies the public keys for the delegate candidate
 const noSignatureDescription =
 	'Creates the transaction without a signature. Your passphrase will therefore not be required.';
 
+const numberOfAccountsDescription =
```

Comment:
This option is specific to the `create account` command, so probably belongs in that file rather than here. Compare with `--output-public-key` in the `encrypt passphrase` command.